### PR TITLE
Support Spring-Boot layered/exploded jar

### PR DESCRIPTION
**Pull request by frehov**
- Add check for existence of exploded/layered jar from spring-boot
- update documentation with mention of layered/exploded spring-boot jars

### CODE REVIEW

File: java/README.md [hunk 1]
```diff
@@ -29,6 +30,28 @@ ENV APP_JAR=my-awesome.jar
 COPY target/my-awesome.jar .
 ```
 
+### Using exploded/layered Spring-Boot JAR
+See the [Spring Boot documentation](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#container-images.dockerfiles) for more information.
+Example provided below for NAV.
+
+```
+FROM eclipse-temurin:<java-version>-jre as builder
```
Comment (albrektsson) [inline java/README.md:38]: Kunne ikke denne vart `FROM ghcr.io/navikt/baseimages:<java-version>-jre as builder` ?

File: java/README.md [hunk 2]
```diff
@@ -29,6 +30,28 @@ ENV APP_JAR=my-awesome.jar
...
+
+WORKDIR application
+COPY target/my-awesome.jar app.jar
+
+# Extract layers
+RUN java -Djarmode=layertools -jar app.jar extract
+
+FROM navikt/temurin:<version>
```
Comment (albrektsson) [inline java/README.md:46]: Se kommentar ovan, annars får du bytte ut denne til `FROM ghcr.io/navikt/baseimages:<java-version>-jre` :)

File: java/README.md (hunk 1 above)
Comment (frehov) [reply java/README.md:38]: helst ikke, siden java/temurin baseimage polluter classpath via CLASSPATH environment variabelen.

File: java/README.md (hunk 2 above)
Comment (frehov) [reply java/README.md:46]: fikser dette

**Review by albrektsson — DISMISSED** [review]

**Review by albrektsson — DISMISSED** [review]

**Review by albrektsson — APPROVED** [review]

**Pull request merged**